pysatMissions/tests/test_plot.py

```diff
@@ -0,0 +1,25 @@
+import pysat
+from pysatMissions import plot as plt
```

Comment:
`plt` is the usual nickname for `matplotlib.pyplot`.  Your use of it as a nickname here could cause confusion.

Reply:
Recommend `mplot`

Reply:
fixed